tests/Mock/Connection/PostBinBroker.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Tests\Mock\Connection;
+
+class PostBinBroker extends TestBroker
+{
+    protected $responseData = '{"message": "Some JSON Response Data"}';
```

Comment:
@DavidGarciaCat This is where I am overriding the property to mock the response from PostBin